app/src/js/actions/index.js

```diff
@@ -128,16 +128,17 @@ export const checkApiVersion = () => {
 };
 
 export const listCollections = (options) => {
+  const { listAll, ...queryOptions } = options;
   return (dispatch, getState) => {
     const timeFilters = fetchCurrentTimeFilters(getState().datepicker);
-    const urlPath = `collections${isEmpty(timeFilters) ? '' : '/active'}`;
+    const urlPath = `collections${isEmpty(timeFilters) || listAll ? '' : '/active'}`;
```

Comment:
Can test that the endless rerender is no longer happening by changing `/active` here to something that doesn't exist

Reply:
I was able to install 1.21.1-alpha.1 and verify this no longer occurs as well.